lib/components/InputBox/BaseInputBox.tsx

```diff
@@ -40,17 +42,49 @@ const BaseInputBox: FC<BaseInputBoxProps> = ({
   ...inputProps
 }) => {
   const [inputID] = useState(inputProps?.id || uniqueid("pebbles__input__"));
+  const [isPassword, setIsPassword] = useState(false);
```

Comment:
this shouldn't be a state variable since it will never change, just use the input type prop directly.

Reply:
I can't, I have to change it from "password" to "text. And I use this to also show the hide/show option in the input. So I couldn't figure out a way to avoid using the state in this case.